All packaging tests failing

Seems to have started yesterday, or late Monday.

https://mc.dot.net/#/product/netcore/30/source/official~2Fdotnet~2Fcorefx~2Frefs~2Fheads~2Fmaster/type/test~2Ffunctional~2Fpackaging~2F/build/20190416.1
through today
https://mc.dot.net/#/product/netcore/30/source/official~2Fdotnet~2Fcorefx~2Frefs~2Fheads~2Fmaster/type/test~2Ffunctional~2Fpackaging~2F/build/20190417.6

there's some like
```
  A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
  Failed to download package 'runtime.win-x86.Microsoft.NETCore.App.3.0.0-preview5-27614-10' from 'https://dotnetfeed.blob.core.windows.net/dotnet-core/flatcontainer/runtime.win-x86.microsoft.netcore.app/3.0.0-preview5-27614-10/runtime.win-x86.microsoft.netcore.app.3.0.0-preview5-27614-10.nupkg'.

```
but there's also some like this (related?)
```
C:\dotnetbuild\work\1d7d4ab2-d77a-4283-a513-58e70840c82f\Work\dd487873-0f74-404f-8a30-ee4c22b674d6\Unzip\netcoreapp3.0\project.csproj : error NU1605: Detected package downgrade: Microsoft.NETCore.Platforms from 3.0.0-preview5.19216.1 to 3.0.0-preview3.19128.7. Reference the package directly from the project to select a different version. 
C:\dotnetbuild\work\1d7d4ab2-d77a-4283-a513-58e70840c82f\Work\dd487873-0f74-404f-8a30-ee4c22b674d6\Unzip\netcoreapp3.0\project.csproj : error NU1605:  project -> Microsoft.NETCore.Platforms.Future 3.0.0-preview5.19216.1 -> Microsoft.NETCore.Platforms (>= 3.0.0-preview5.19216.1) 
C:\dotnetbuild\work\1d7d4ab2-d77a-4283-a513-58e70840c82f\Work\dd487873-0f74-404f-8a30-ee4c22b674d6\Unzip\netcoreapp3.0\project.csproj : error NU1605:  project -> Microsoft.NETCore.Platforms (>= 3.0.0-preview3.19128.7)

```

@joperezr @ericstj

--- a/pkg/test/frameworkSettings/netcoreapp3.0/settings.targets
+++ b/pkg/test/frameworkSettings/netcoreapp3.0/settings.targets
@@ -4,6 +4,7 @@
     <NETCoreAppMaximumVersion>3.0</NETCoreAppMaximumVersion>
     <!-- use the most recent MS.NETCore.App we have from upstack -->
     <RuntimeFrameworkVersion>$(MicrosoftNETCoreAppPackageVersion)</RuntimeFrameworkVersion>
+    <DisableImplicitNETCorePlatformsReference>true</DisableImplicitNETCorePlatformsReference>
   </PropertyGroup>
 
   <ItemGroup>

